src/components/Dashboard.tsx

```diff
@@ -0,0 +1,158 @@
+import React from 'react'
+import { Link } from 'react-router-dom'
+import { Header, Button, Icon, SemanticCOLORS, SemanticICONS } from 'semantic-ui-react'
+import { useUserState } from '../contexts/UserState'
+import constants from '../utils/constants'
+import strings from '../utils/constants'
+import { USER_ROLES } from '../utils/data'
+import { PermissionPolicyType, Filter } from '../utils/generated/graphql'
+import useListApplications from '../utils/hooks/useListApplications'
+import useListTemplates from '../utils/hooks/useListTemplates'
+import usePageTitle from '../utils/hooks/usePageTitle'
+import { TemplateDetails, TemplateInList } from '../utils/types'
+
+const Dashboard: React.FC = () => {
+  const {
+    userState: { templatePermissions, isNonRegistered },
+    logout,
+  } = useUserState()
+  const {
+    templatesData: { templatesByCategory },
+  } = useListTemplates(templatePermissions, false)
+  if (isNonRegistered) {
+    logout()
+    return null
+  }
+
+  usePageTitle(strings.PAGE_TITLE_HOME)
+
+  return (
+    <div id="dashboard">
+      {templatesByCategory.map(
+        ({ templates, templateCategory: { categoryIcon, categoryTitle } }) => (
+          <div key={categoryTitle} className="template-category">
+            <div className="title">
+              {categoryIcon && <Icon size="large" color="grey" name={categoryIcon} />}
+              <Header as="h4">{categoryTitle}</Header>
+            </div>
+            <div className="templates">
+              {templates.map((template) => (
+                <TemplateComponent key={template.code} template={template} />
+              ))}
+            </div>
+          </div>
+        )
+      )}
+    </div>
+  )
+}
+
+const TemplateComponent: React.FC<{ template: TemplateInList }> = ({ template }) => {
+  const { name, code, hasApplyPermission, filters } = template
+  return (
+    <div className="template">
+      <div className="title">
+        <Header as="h5">{name}</Header>
+
+        {hasApplyPermission && (
+          <Button inverted size="small" as={Link} to={`/application/new?type=${code}`} primary>
+            {constants.BUTTON_DASHBOARD_NEW}
+          </Button>
+        )}
```

Comment:
Would there be another one for `hasNonApplyPermission`?

Reply:
not sure what you mean, this is to control the new application button.

Reply:
Nice. Ii was just asking if you have created also the variable `hasNonApplyPermission` to be used in a similar way to go straight to a review page for example. I haven't seen that, maybe it's there.